.github/workflows/link_checkPR.yml

```
@@ -0,0 +1,63 @@
+#Checks links in a PR to ensure they are valid. If link is valid but failing, it can be added to the .lycheeignore file
+#Code source: https://github.com/lycheeverse/lychee-action/issues/238
+
+name: link check on PR
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+
+      - name: Check out main branch
+        run: git checkout main
+
+      - name: Dump all links from main
+        id: dump_links_from_main
+        uses: lycheeverse/lychee-action@v1
```

Anything wrong with using `v2` version?
```suggestion
        uses: lycheeverse/lychee-action@v2
```